test/handlers/s3.test.ts

```diff
@@ -20,7 +20,7 @@ describe('s3', () => {
       secretAccessKey: 'DERP',
     });
 
-    const ctx = helpers.getCtx();
```

Comment:
here is the proof the test toolchain now supports TS

Reply:
I tried switching the helper Context object to TS but it has some confusing behaviours on whether the headers are a Map or an object. But this is the beauty of TS is that we can fix this.